src/app/page.tsx

```
@@ -208,8 +228,8 @@ export default function Home() {
           <Alert.Title>Change device language</Alert.Title>
           <Alert.Description>
             Before starting the process, it is recommended to change the device
-            language to English. To do this, select “Settings” icon, then click
-            “OK / Confirm” button and “OK / Confirm” again until English is
+            language to English. To do this, select “Settings" icon, then click
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)